rir/src/interpreter/interp.cpp

```diff
@@ -3526,3 +3513,4 @@ SEXP rirEval_f(SEXP what, SEXP env) {
 
     assert(false && "Expected a code object or a dispatch table");
 }
+}
```

Comment:
`// namespace rir`

Reply:
bah. I wonder why sometimes clang-format does that for me and sometimes not :(